packages/components-vue/vite.config.ts

```diff
@@ -0,0 +1,38 @@
+import { defineConfig } from 'vite';
```

Comment:
Rename to`vite.config.lib.ts` to match tsconfig and improve clarity.

Reply:
I would not add lib to the root tsconfig and the default vite.config as we have not done it in angular & react. Also adding lib would require to add a path to the designated vite.config and tsconfig files for the `vite build` & `vue-tsc` scripts?